static/swagger-specs/unified-tags.yaml

```diff
@@ -0,0 +1,983 @@
+  /tagCategory:
+    get:
+      tags:
+        - Tag categories
+      summary: List tag categories
+      operationId: listTagCategories
+      parameters:
+        - $ref: '#/components/parameters/start'
+        - $ref: '#/components/parameters/limitTagCategories'
+        - $ref: '#/components/parameters/propertyTagCategories'
+        - $ref: '#/components/parameters/sortByTagCategories'
+        - $ref: '#/components/parameters/sortOrderTagCategories'
+        - $ref: '#/components/parameters/authorization'
+        - $ref: '#/components/parameters/content-type'
+        - $ref: '#/components/parameters/x-gw-ims-org-id'
+        - $ref: '#/components/parameters/x-api-key'
+      responses:
+        200:
+          x-summary: "Success"
+          description: "A list of tag categories that belongs to the organization is returned."
+          content: 
+            application/json:
+              schema: 
+                $ref: "#/components/schemas/tagCategoriesList"
+    post: 
+      tags: 
+        - Tag categories
+      summary: "Create a new tag category"
+      operationId: createTagCategory
+      parameters: 
+        - $ref: '#/components/parameters/authorization'
+        - $ref: '#/components/parameters/content-type'
+        - $ref: '#/components/parameters/x-gw-ims-org-id'
+        - $ref: '#/components/parameters/x-api-key'
+      requestBody: 
+        description: "The request body to create the tag category."
+        content: 
+          "*/*":
+            schema: 
+              $ref: "#/components/schemas/createTagCategoryBody"
+        required: true
+      responses: 
+        200:
+          x-summary: "Success"
+          description: "The tag category was successfully created."
+          content: 
+            application/json:
+              schema: 
+                $ref: "#/components/schemas/tagCategory"
+  /tagCategory/{TAG_CATEGORY_ID}:
+    get: 
+      tags: 
+        - Tag categories
+      summary: Retrieve a specific tag category
+      operationId: retrieveTagCategory
+      parameters: 
+        - $ref: '#/components/parameters/tag-category-id'
+        - $ref: '#/components/parameters/authorization'
+        - $ref: '#/components/parameters/content-type'
+        - $ref: '#/components/parameters/x-gw-ims-org-id'
+        - $ref: '#/components/parameters/x-api-key'
+      responses: 
+        200:
+          x-summary: Success
+          description: "The specified tag category was successfully retrieved."
+          content: 
+            application/json:
+              schema: 
+                $ref: "#/components/schemas/tagCategory"
+    patch: 
+      tags: 
+        - Tag categories
+      summary: Update a specific tag category
+      description: "Only the system administrator and the product administrator can use this API call."
+      operationId: updateTagCategory
+      parameters: 
+        - $ref: '#/components/parameters/tag-category-id'
```

Comment:
You cannot use this reference as it says: "The ID of the tag category you're retrieving."
but it should say:
"The ID of the tag category you're updating."